### PR TITLE
Instructions for installing command completion

### PR DESCRIPTION
**Pull request by dylanPowers**
I have instructions for installing command completion in Linux bash. If anyone would like to add instructions for other shells and OSX. That would be great.

### CODE REVIEW

**Comment by whyrusleeping** [comment]
This looks good to me, only comment would be maybe adding a link here: https://www.debian-administration.org/article/316/An_introduction_to_bash_completion_part_1

for "further reading"

**Comment by dylanPowers** [comment]
@whyrusleeping I went ahead and added in that reference

File: docs/command-completion.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+is only temporary and to fully enable it, you'll have to follow one of the steps
+below.
+
+### Bash on Linux
+For bash, completion can be enabled in a couple of ways. One is to add
+```bash
+if [ -n "$GOPATH" ]; then
+  source $GOPATH/src/github.com/ipfs/go-ipfs/misc/completion/ipfs-completion.bash
+fi
+```
```
Comment (jbenet) [inline docs/command-completion.md]: should tell the user what to do if they _dont_ have go. it's possible that we could make a simple `ipfs-install-bash-completion` tool. i want to start making tools that we can deploy through ipfs itself, whose hashes are hardcoded in the binary.

**Comment by jbenet** [comment]
@dylanPowers :+1: LGTM except the last comment above o/

**Comment by dylanPowers** [comment]
Hmm, I'm trying to think of a good place for the completion script to be stored. I suppose we could instruct them to copy it into the ~/.ipfs

**Comment by whyrusleeping** [comment]
@dylanPowers shouldnt it go into /etc/completions?

**Comment by dylanPowers** [comment]
@whyrusleeping This is for sourcing from `~/.bash_completion`. If the user is going to put it anywhere globally, they might as put it in `/etc/bash_completion.d/` and not even bother with `~/.bash_completion`

**Comment by whyrusleeping** [comment]
ah, gotcha. putting it in the .ipfs dir makes sense. although, i dont use bash completions, and fish does its own thing in `$HOME/.config/fish`

**Comment by dylanPowers** [comment]
Can fish read bash completion scripts, or does it need it's own script?

**Comment by dylanPowers** [comment]
I committed that last change to remove the Go dependency from the instructions

**Comment by whyrusleeping** [comment]
LGTM, RFM

**Pull request merged**